src/OrleansSQLUtils/Storage/DbStoredQueries.cs

```diff
@@ -192,14 +192,18 @@ internal static ReminderEntry GetReminderEntry(IDataRecord record)
             /// </summary>
             private static string TryGetSiloName(IDataRecord record)
             {
-                for (var i = 0; i < record.FieldCount; ++i)
```

Comment:
I see. I personally have no problem with it looking for the column in case sensitive way. But I'm OK with your change. This column is different because we've (re)added it and wanted to keep the code backwards compatible.